File: ProjetoDev/F1/Models/Torneio.cs
namespace F1.Models;

public class Torneio
{
    public int id { get; set; }
    public string? ano { get; set; }
    public int numCorridas { get; set; }
    public int vencedorID { get; set; }
    public Piloto? vencedor { get; set; }
    public int EquipeVencedoraID { get; set; }
    public Equipe? EquipeVencedora { get; set; }
    public DateTime criadoEm { get; set; } = DateTime.Now;
}

File: frontf1/src/models/Torneio.ts
import {Piloto} from "./Piloto";
import {Equipe} from "./Equipe";

export interface Torneio{
    
    id: number;
    ano: string;
    numCorridas: number;
    vencedorID: number;
    vencedor: Piloto;
    equipeVencedoraID: number;
    equipeVencedora: Equipe;
    
}